src/Page/Textile.elm

```diff
@@ -754,17 +754,17 @@ productCategoryField { products } query =
                 |> Result.withDefault default
 
         autocompleteState =
-            AutocompleteSelector.init (nameFromProductId "") (List.map .id products)
+            products
+                |> List.sortBy .name
+                |> List.map .id
+                |> AutocompleteSelector.init (nameFromProductId "")
     in
-    div [ class "row align-items-center g-2" ]
-        [ label
-            [ for "selector-product"
-            , class "col-sm-4 col-form-label text-truncate"
-            ]
+    div [ class "d-flex flex-column" ]
+        [ label [ for "selector-product", class "form-label text-truncate" ]
             [ text "Catégorie" ]
         , button
-            [ class "col-sm-8 flex-fill form-select ElementSelector text-start w-auto"
-            , id "selector-product"
+            [ id "selector-product"
+            , class "form-select ElementSelector text-start w-auto"
```

Comment:
Just for my understanding, this TitleCase (`ElementSelector`) looks strange from the outside, why not use the same style with `-` than the other classes?

Reply:
It's the ongoing convention for custom elements in styles.scss